sphinx_immaterial/dropdown.py

```diff
@@ -0,0 +1,78 @@
+"""Taken from sphinxcontrib-details
```

Comment:
If this stays as is, please retain the original copyright notice of the source code from which it was derived, and modify the LICENSE file to include the relevant license.

Reply:
It uses [Apache v2](https://github.com/tk0miya/sphinxcontrib-details-directive/blob/master/LICENSE). Again, this module could be removed if I can remove the mkdocs-material CSS that relies on a md ext (something we can't use anyway).

Reply:
I added the complete copy of the license in dropdown.py. Additionally, I appended the copyright notice to the LICENSE file. Per the Apache v2 stipulations, I have documented (in source code comments) what was modified in dropdown.py.